examples/workflows/global-config.yaml

```diff
@@ -11,12 +11,17 @@ spec:
       command:
       - cowsay
       image: docker/whalesay:latest
-  - name: 'say'
+  - name: say
     script:
-      command: ['python']
-      image: 'image-say'
-      source: |
-        import os
+      command:
+      - python3
+      image: image-say
+      source: 'import os
```

Comment:
Is there an extra space in the script str generation step? Perhaps we could remove it?